app/src/pages/datasets/DatasetsTable.tsx

```diff
@@ -157,6 +165,21 @@ export function DatasetsTable(props: DatasetsTableProps) {
             <DatasetActionMenu
               datasetId={row.original.id}
               datasetName={row.original.name}
+              datasetDescription={row.original.description}
+              datasetMetadata={row.original.metadata}
+              onDatasetEdit={() => {
+                notifySuccess({
+                  title: "Dataset updated",
+                  message: `${row.original.name} has been successfully updated.`,
+                });
+                refetch({}, { fetchPolicy: "store-and-network" });
```

Comment:
this totally works. But might make sense to just switch this stuff to relay refetching of fragments. makes things a bit more encapsulated.

Reply:
yeah i wanted to spread the fragment. Actually is this possible without changing mutations server side since datasets are queryable top level? i didn't consider that, i was thinking i needed the parent on the return of the Created / Edited dataset, not sure how flexible relay is with that... will give it a shot

Reply:
I think you might have to. Let's file a follow-up and do it. Best to make things more efficient as we go.

Reply:
yeah i'll file a follow up can't query datasets off the return

Reply:
https://github.com/Arize-ai/phoenix/issues/4008